Rules/CTF/Scripts/CTF_GiveSpawnItems.as

```diff
@@ -267,32 +230,101 @@ void onRender(CRules@ this)
 {
 	if (g_videorecording || this.isGameOver())
 		return;
-
+	
 	CPlayer@ p = getLocalPlayer();
-	if (p is null || !p.isMyPlayer()) { return; }
-
-	string propname = getCTFTimerPropertyName(p);
+	if (p is null || !p.isMyPlayer()) return;
```

Comment:
this check is not needed, safe to remove

(this is called once client side per frame, p will always be my player, and p will always exist as long as im in game)

Reply:
this is vanilla code, but removed

~~if this will cause dumb issues in the future i will headbonk you~~